### ISSUE TITLE
Arduino Examples opens 6 times

### ISSUE THREAD

**Issue by driconmax** [microsoft/vscode-arduino#1070]
Every time i upload the code to the Arduino, at the end, 6 instances of the "Arduino Examples" window show up, and 1 extra in the other split screen "Preview README.md" also does.

**Comment by NanoCodeBug** [comment]
I'm experiencing the same issue when I have usb detection enabled for the arduino plugin. Disabling it and restarting vscode prevents it from opening the examples and readme.

**Comment by hellyzh** [comment]
Yes, adding `"arduino.enableUSBDetection": false` to vscode preferences settings.json can disable the usb detection.
